src/aatoolbox/config/countries/bfa.yaml

```diff
@@ -3,3 +3,7 @@ codab:
   hdx_dataset_name: bfa_adm_igb_20200323_SHP.zip
   layer_base_name: bfa_admbnda_adm{admin_level}_igb_20200323.shp
   admin_level_max: 3
+fewsnet:
+  region_name: west-africa
+usgs_ndvi:
+  region_name: west-africa
```

Comment:
Ha tests are gonna save you here. But now getting an error as this par is named `region_name` while countryconfig is expecting `area_name`

Reply:
Ha yes, haven't tested this stuff yet in new, will do. Thanks for catching up!